packages/frontend/src/components/icons/symbols/CanonicalIcon.tsx

```diff
@@ -3,6 +3,10 @@ import React, { SVGAttributes } from 'react'
 
 import { Icon } from '../Icon'
 
+/*  IMPORTANT
+  If you change this file you need to update the following file too:
+  * packages/frontend/src/scripts/charts/styles.ts
```

Comment:
Provide more context on which icon should be changed as `styles.ts` file does not tell which one should change too